src/sdk/namespace_fs.js

```diff
@@ -1104,6 +1104,29 @@ class NamespaceFS {
             throw this._translate_object_error_codes(err);
         }
     }
+
+    async check_access(fs_account_config, dir_path) {
+        try {
+            dbg.log0('check_access: dir_path', dir_path, 'fs_account_config', fs_account_config);
+            await nb_native().fs.checkAccess(fs_account_config, dir_path);
```

Comment:
A bit weird that we are calling open on directories and not opendir, but it seems to work